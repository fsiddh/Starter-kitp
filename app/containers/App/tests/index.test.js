import React from 'react'; // eslint-disable-line no-unused-vars
import renderer from 'react-test-renderer';
import App from '../index';

describe('<App />', () => {
  it('should render some routes', () => {
    const renderedComponent = renderer.create(<App />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
