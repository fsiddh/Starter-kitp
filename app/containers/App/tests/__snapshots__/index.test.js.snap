// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render some routes 1`] = `<div />`;
